setup.cfg

```diff
@@ -7,8 +7,8 @@ project_urls =
     GitHub = https://github.com/colcon/colcon-gradle/
 author = Esteve Fernandez
 author_email = esteve@apache.org
-maintainer = Esteve Fernandez
-maintainer_email = esteve@apache.org
+maintainer = Jacob Perron
```

Comment:
I thought you meant to add you as a maintainer in addition to me, not replace. Is there a way to add both?

Reply:
Yeah, sorry, I think we can add both as a comma separate list. See 67df0fc.
Here's how the package is rendered on the test PyPI instance: https://test.pypi.org/project/colcon-gradle/0.0.3/